webapp/src/index.js

```diff
@@ -66,11 +66,15 @@ export default class Plugin {
             return frontendListName;
         };
 
-        const refresh = ({data: {lists}}) => lists.forEach((listName) => store.dispatch(list(false, getFrontendListName(listName))));
+        const refresh = ({data: {lists}}) => {
```

Comment:
Unrelated to this PR, but I would like to make this a bit safer since we're editing this function:

```suggestion
        const refresh = (payload) => {
            if (payload.data && payload.data.lists) {
```